rollup/init.cjs

```diff
@@ -20,4 +20,14 @@ export default document => ${
     .replace(/^(\s+)replaceWith\(([^}]+?)\}/m, '$1/* c8 ignore start */\n$1replaceWith($2}\n$1/* c8 ignore stop */')
     .replace(/^(\s+)(["'])use strict\2;/m, '$1$2use strict$2;\n\n$1const { constructor: DocumentFragment } = document.createDocumentFragment();')
     .replace(/^[^(]+/, '')
+    .replace(/\n  exports\.Hole = Hole;[\S\s]*return exports;/m, `
```

Comment:
what has this to do with this MR?

Reply:
It fixes type hints of `uhtml/init`.

Reply:
OK ... then we need to do a bit more ... to start with, the `exports` object is not needed, we need to replace that too. I now need to maintain manually all exports in this file + it's not clear once we remove `tsc` what is typescript dependency doing in here at all ... this feels like a breaking change too ... thoughts?

Reply:
also this line feels particularly cumbersome `render : keyed$1,`

Reply:
uhm ... I think this is landing in here with better logic to let me publish more than just `init.js` there: https://github.com/WebReflection/uhtml/pull/114